pkg/ocm/ocm.go

```diff
@@ -21,15 +21,15 @@ var (
 	sl_clusterHasGoneMissing = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/cluster_has_gone_missing.json"
 )
 
-type ocmClient struct {
+type OcmClient struct {
 	conn *sdk.Connection
```

Comment:
If you export OcmClient would it make sense to export Conn and Cfg as well?

Reply:
Added a comment, feel free to TAL